src/vmm/src/arch/aarch64/vcpu.rs

```diff
@@ -0,0 +1,292 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Portions Copyright 2017 The Chromium OS Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the THIRD-PARTY file.
+
+use std::path::PathBuf;
+
+use kvm_bindings::*;
+use kvm_ioctls::VcpuFd;
+use utils::vm_memory::GuestMemoryMmap;
+
+use super::get_fdt_addr;
+use super::regs::*;
+
+/// Errors thrown while setting aarch64 registers.
+#[derive(Debug, PartialEq, Eq, thiserror::Error)]
```

Comment:
It could ease the review process if the refactoring and code movements were extracted into separate commit(s).

Reply:
I can split it into 2 commits, but for now, while there are changes to be made, I would prefer not to do it.

Reply:
I don't insist, but since some more review iterations are expected, every round would take less time if reviewers only have to look at functional changes.

Reply:
I guess your biggest concern is that git shows that `regs.rs` has a bunch of changes, even thought it only contain new code, and `vcpu.rs` as new file, even though it is just old `regs.rs`, but renamed.

Reply:
yes, I guess so.

Reply:
I renamed `regs.rs` into `registers.rs` and now git shows changes correctly.